packages/ui-kit/src/components/Leaderboard/Leaderboard.tsx

```diff
@@ -122,6 +74,31 @@ export function Leaderboard(props: LeaderboardProps) {
         }
       }
 
+      const customBarLabel: Plugin = {
```

Comment:
This change is a proposal for a new look for `Bar` variant and has not yet been confirmed.
<img width="887" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/c2af9ed4-02be-46f6-ac5c-2ad964d3f9b5">